daemon/response.go

```diff
@@ -72,14 +73,31 @@ func (r *resp) transmitMaintenance(kind errorKind, message string) {
 	}
 }
 
+func (r *resp) addWarningsToMeta(ws func() (int, time.Time)) {
```

Comment:
not sure I understand why does it take a function and not count, stamp ?

Reply:
I'll change it to do this. Whatever the reason was it seems silly.